src/olympia/editors/tests/test_utils.py

```diff
@@ -216,6 +217,9 @@ def test_no_request(self):
         helper = self.get_helper()
```

Comment:
maybe add `content_review_only=False` to this one to be explicit

Reply:
I left it without the argument because all other tests also call `get_helper()` without the `content_review_only` arg and I didn't want to modify ALL the tests. To be safe though I just added an assertion making sure the property is what we expect it to be when no argument is passed.